example/serializers.py

```diff
@@ -50,6 +58,25 @@ class Meta:
         meta_fields = ('copyright',)
 
 
+# DRF default serializer to test default DRF functionalities
+class BlogDRFSerializer(serializers.ModelSerializer):
```

Comment:
shouldn't this be derived from drf_serilazers.ModelSerializer as well? Also best move the comment into the class a block comment.